tests/test_mssqlcliclient.py

```diff
@@ -202,6 +203,37 @@ def test_stored_proc_multiple_result_sets(self):
         finally:
             shutdown(client)
 
+    
+    def test_select_result_unicode_encoded(self):
+        """
+            Verify the column names and string values in rows returned by select statement are properly encoded as unicode.
+        """
+        db_name = 'db_' + random_str()
```

Comment:
>db_ [](start = 19, length = 3)

let's add the local machine name and mssqlcli to the database name. We use a shared server, so this will give folks some idea who is creating this database.

Reply:
Added machine name and 'mssqlcli' string in the table name as requested.